src/encoded/schemas/changelogs/biosample.md

```diff
@@ -4,7 +4,7 @@
 
 * Added *post_differentiation_time* and *post_differentiation_time_units* properties.
 * Added *perturbed* calculated property to indicate if the biosample has been perturbed using a treatment or a genetic modification for non-tagging purposes.
-* Added *post_nucleic_acid_delivery_time* and *post_nucleic_acid_delivery_time_units* properties.
+* Added *post_nucleic_acid_delivery_time* and *post_nucleic_acid_delivery_time_units* properties and to the calculated property of summary.
```

Comment:
Shouldn't ENCD-5139 had done this?

Reply:
ENCD-5139 doesn't add the properties to the biosample_summary. It only created the properties. ENCD-5157 ticket does 2 things. 1) display post_nucleic_acid_delivery_time and post_nucleic_acid_delivery_time_units on the biosample page, 2) add post_nucleic_acid_delivery_time and post_nucleic_acid_delivery_time_units to calculated biosample_summary. Let me know if this makes sense.